content/sensu-go/5.21/reference/backend.md

```diff
@@ -307,6 +307,8 @@ General Flags:
       --log-level string                    logging level [panic, fatal, error, warn, info, debug] (default "warn")
       --pipelined-buffer-size int           number of events to handle that can be buffered (default 100)
       --pipelined-workers int               number of workers spawned for handling events through the event pipeline (default 100)
+      --require-fips
+      --require-openssl
```

Comment:
```suggestion
      --require-fips                            indicates whether fips support should be required in openssl
      --require-openssl                     indicates whether openssl should be required instead of go's built-in crypto
```